app/views/pruebas/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class='columns'>
+    <div class='column'></div>
+    <div class='column'>
+        <h1 class="title is-1">Editar</h1>
+        <%= render 'form', prueba: @prueba %>
+    </div>
+    <div class='column'></div>
```

Comment:
Avoid empty containers.
If you want the second container to be in the center, then apply `justify-content: center` on its parent and remove this useless columns.
Here a resource:
https://css-tricks.com/snippets/css/a-guide-to-flexbox/